src/Core/src/PropertyMapperExtensions.cs

```diff
@@ -145,5 +145,80 @@ void newMethod(IElementHandler handler, IElement view)
 				action?.Invoke(handler!, view);
 			});
 		}
+
+		/// <summary>
+		/// Used internally to provide a fluent API for easily modifying
+		/// mappers at the point of instantiation.
+		/// Specify a method to be run before an existing property mapping.
```

Comment:
```suggestion
		/// Specify a method to be run instead of an existing property mapping.
		/// This can be use to override a mapping entirely, and will NOT fall back to any existing mappers.
```